public/editor/FeatureGallery.tsx

```diff
@@ -255,28 +258,36 @@ const FeatureSelector = ({
             // @ts-expect-error TS doesnt like conditional array vs string
             hasMultipleColors ? selectedVal[colorIndex] : selectedVal;
 
+          let presetColors = hasMultipleColors
+            ? gallerySectionConfig.renderOptions.valuesToRender.map(
+                (colorList: string[]) => colorList[colorIndex],
```

Comment:
I'll fix this when I merge in a minute, but there's no need for this type annotation, TypeScript already knows it's string[]